sdk/storage/azblob/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
I think this title should actually be **Azure Blob Storage client module for Go**. I see that other SDKs call it "Azure Storage Blobs" but the service is called "Azure Blob Storage." Thoughts?

Reply:
I think it's better to be consistent across SDKs. I think we've also had this discussion about Tables in the past.

Reply:
I talked to Kyle and he linked me to our style guide which says to use "Azure Blob Storage" so we should use that. And we'll work on getting the other SDKs updated to follow the same convention since we definitely want to be consistent.